src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala

```diff
@@ -41,16 +47,25 @@ final case class AuthorizationApiServiceImpl(
 
   private val admittedSessionClaims: Set[String] = Set(UID, ORGANIZATION, NAME, FAMILY_NAME, EMAIL)
 
+  // TODO move me in commons
+  private val SELFCARE_ID_CLAIM = "selfcare_id"
```

Comment:
It needs to be added [with this PR](https://github.com/pagopa/interop-commons/pull/200) to commons too and to use its getter